src/config.ts

```diff
@@ -1,29 +1,29 @@
-import * as fs from 'fs'
-import YAML from 'yaml'
-import {ConfigurationOptions, ConfigurationOptionsSchema} from './schemas'
-import path from 'path'
+import * as core from '@actions/core'
+import {
+  ConfigurationOptions,
+  ConfigurationOptionsSchema,
+  Severity
+} from './schemas'
 
-export const CONFIG_FILEPATH = './.github/dependency-review.yml'
+export function readConfig(): ConfigurationOptions {
+  let options: ConfigurationOptions = {}
 
-export function readConfigFile(
-  filePath: string = CONFIG_FILEPATH
-): ConfigurationOptions {
   // By default we want to fail on all severities and allow all licenses.
-  const defaultOptions: ConfigurationOptions = {
-    fail_on_severity: 'low'
-  }
+  const severity = core.getInput('fail-on-severity') || 'low'
```

Comment:
I don't like having to do the OR here, and I'd expect Zod to take care of this, but removing the clause returns `undefined`.